### DISCUSSION TITLE
Need to make dialog draggable

### DISCUSSION THREAD

**Discussion by jduncanRadBlue** [RocketCommunicationsInc/astro#888] [Feature Requests]
Is there a way to make the dialog component moveable/draggable?

**Comment by micahjones13** [comment]
Hey @jduncanRadBlue, after some testing it does not seem like our current implementation of rux-dialog can be draggable. I'm going to convert this issue into a feature request so we can start to explore adding in this functionality. 
Thanks for raising this, I'll keep updates on our progress on the feature request discussion.
